package.json

```
@@ -54,6 +54,11 @@
     "isomorphic-unfetch": "^3.1.0",
```

I haven't dug too deep into the issue, but "Up to (excluding) 3.1.1" makes me think we should also bump our direct dependency. But I may just be misunderstanding the override system with multiple versions of the same dependency around.

```suggestion
    "isomorphic-unfetch": "^3.1.1",
```

---

I took a look into `node-fetch` and the reported issue isn't fixed in the latest - opened https://github.com/node-fetch/node-fetch/pull/1599